### PR TITLE
test: stabilize a couple of tests

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: src/components/menu/menu/menu.spec.ts [hunk 1]
```diff
@@ -1,76 +1,73 @@
...
+            <sbb-block-link href="https://www.sbb.ch/en">Profile</sbb-block-link>
+            <sbb-menu-button icon-name="tick-small">View</sbb-menu-button>
+            <sbb-menu-button icon-name="pen-small" amount="1" disabled>Edit</sbb-menu-button>
+            <sbb-menu-button icon-name="swisspass-small" amount="2">Details</sbb-menu-button>
+            <sbb-divider></sbb-divider>
+            <sbb-menu-button icon-name="cross-small">Cancel</sbb-menu-button>
+          </sbb-menu>
+        </div>
+      `);
+      await waitForLitRender(testFixture);
```
Comment (kyubisation) [inline src/components/menu/menu/menu.spec.ts:33]: Can be removed

File: src/components/option/optgroup/optgroup.spec.ts [hunk 2]
```diff
@@ -16,16 +16,18 @@ describe(`sbb-optgroup`, () => {
...
+                <sbb-option value="1">1</sbb-option>
+                <sbb-option value="2">2</sbb-option>
+              </sbb-optgroup>
+            </sbb-autocomplete>
+            <div id="anchor"></div>
+          </div>
         `);
-        elem = document.querySelector('sbb-optgroup')!;
+        elem = testFixture.querySelector('sbb-optgroup')!;
         await waitForLitRender(elem);
```
Comment (kyubisation) [inline src/components/option/optgroup/optgroup.spec.ts:31]: Can be removed

File: src/components/option/optgroup/optgroup.spec.ts [hunk 3]
```diff
@@ -56,16 +58,18 @@ describe(`sbb-optgroup`, () => {
...
```
Comment (kyubisation) [inline src/components/option/optgroup/optgroup.spec.ts:73]: Can be removed

**Review by kyubisation — APPROVED** [review]
LGTM 👍

File: src/components/core/testing/private/a11y-tree-snapshot.ts [hunk 4]
```diff
@@ -45,9 +45,10 @@ export function testA11yTreeSnapshot(
       await a11yTreeEqualSnapshot();
     });
 
-    testIf(isSafari() && !exclude.safari, 'Safari', async () => {
-      await a11yTreeEqualSnapshot();
-    });
+    // TODO: Try if tests on CI are no longer flaky
+    // testIf(isSafari() && !exclude.safari, 'Safari', async () => {
+    //   await a11yTreeEqualSnapshot();
+    // });
```
Comment (TomMenga) [inline src/components/core/testing/private/a11y-tree-snapshot.ts:51]: Just a nitpick, we can also remove 'safari' from the `exclude` param at line 34

**Review by TomMenga — APPROVED** [review]
LGTM 👍 

Just a nitpick

File: src/components/core/testing/private/a11y-tree-snapshot.ts (hunk 4 above)
Comment (jeripeierSBB) [reply src/components/core/testing/private/a11y-tree-snapshot.ts:51]: Asked @kyubisation and we would like to keep to option for the moment

**Pull request merged**